src/cmd/lium.bash

```diff
@@ -141,7 +141,7 @@ _lium_get_options() { # current
 _lium() { # command current prev
   local cur=$2
   local prev=$3
-  local dir_opts="--repo --dir --dest"
+  local dir_opts="--repo --dir --dest --cros --arc"
```

Comment:
qq: Do we still have `--repo` usage?

Reply:
If not, removing --repo from here is a good idea!

Reply:
Sure, I also removed `--dir` and `--dest` options since they don't exist.

Reply:
https://github.com/google/lium/blob/6a5d9950c58bda47b6de88b1d1bf3bf0d90b0203/src/cmd/dut.rs#L115
actually `--dest` is still used 😇

Reply:
Sorry... I missed them.